openedx/core/djangolib/tests/utils.py

```diff
@@ -0,0 +1,61 @@
+from django.core.cache import caches
+from django.test import TestCase, override_settings
+
+
+class CacheSetupMixin(TestCase):
```

Comment:
Thinking on this class a little more, maybe it would be better to make this something that inherits from `object` and has a handful of utility functions in it (e.g. init, clear, etc.). That way things using it could be more selective about the cache behavior. For instance, in the case of `SharedModuleStoreTestCase`, we probably don't want to wipe the `mongo_metadata_inheritance` with each test `setUp` and `tearDown`, because we're only setting up the modulestore stuff once and using it across test methods.